### PR TITLE
Add GI GTK/GDK/Cairo standalone test

### PR DESCRIPTION
**Pull request by hacklschorsch**
# What does this PR do?

This adds a regression test for 153c0463be and PR #2485.
It is almost identical with the [reproducer @kayhayen posted for #2471](https://github.com/Nuitka/Nuitka/issues/2471#issuecomment-1744525538).

I do not know what file preamble (copyright notices), meta tags (like ignoring warnings, configuring isort...) or changes to dependecy definitions (to make sure the test runners actually have Gtk, Gdk and Cairo) are required unfortunately.
Happy to add if given pointers, but of course please feel free to edit this as you see fit.

# Why was it initiated? Any relevant Issues?

Issue #2471 triggered my looking into this.

# PR Checklist

- [x] Correct base branch selected? Should be `develop` branch.
- [ ] Enabled commit hook or executed `./bin/autoformat-nuitka-source`.
  - Sorry, again used black manually.
- [ ] All tests still pass. Check the Developer Manual about `Running the Tests`. There are GitHub
  Actions tests that cover the most important things however, and you are welcome to rely on those,
  but they might not cover enough.
  - I cannot run them on my setup still :(
- [ ] Ideally new features or fixed regressions ought to be covered via new tests.
  - This is trying to help with that - providing a test for an already fixed regression.
- [ ] Ideally new or changed features have documentation updates.

### CODE REVIEW

**Comment by kayhayen** [comment]
It needs a `# nuitka-skip-unless-imports: cairo` or else it's going to fail on GitHub, where it's not installed. We currently only do virtualenv based tests in Nuitka-Watch, where I may add it once it supports MSYS2 as well, i.e. flavors.

**Comment by hacklschorsch** [comment]
No need to have Cairo here really - the test would be almost as useful without it, no?

**Comment by kayhayen** [comment]
You can of course make the `cairo` usage optional, but then `gi` is also not going to be installed everywhere, so lets not do it here. It's ok if the test is skipped in most environments, if private CI has that, or if it'S also added to Nuitka-Watch which I intend to do.

File: tests/standalone/GiUsing.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+import cairo
+import gi
+
+gi.require_version("Gtk", "3.0")
+gi.require_version("Gdk", "3.0")
+
+from gi.repository import Gdk, Gtk
+
+# isort:start
```
Comment (kayhayen) [inline tests/standalone/GiUsing.py:11]: It would appear that the isort start  moniker is something you wanted before the gi.repository import?

**Pull request merged**